tests/unittests/distros/test_netconfig.py

```diff
@@ -1298,7 +1315,7 @@ def test_mariner_network_config_v1_with_duplicates(self):
 
 
 def get_mode(path, target=None):
-    return os.stat(subp.target_path(target, path)).st_mode & 0o777
+    return os.stat(subp.target_path(target, path)).st_mode & 0o7777
```

Comment:
`st_mode & 0o7777` is a noop right?

Reply:
ohhh right. yeah this'll just return the st_mode and any sticky bits etc.... should be dropped as some of our tests want those sticky bits.

Reply:
I forgot about the need to mask st_mode flags like [S_IFREG  = 0o100000](https://github.com/python/cpython/blob/3.11/Lib/stat.py#L39). Leaving the mask intact.